src/plugins/kibana_react/public/table_list_view/table_list_view.tsx

```diff
@@ -450,34 +492,10 @@ class TableListViewUi extends React.Component {
         className="itemListing__page"
         restrictWidth
       >
-        <EuiPageBody>
-          {this.renderPageContent()}
-        </EuiPageBody>
+        <EuiPageBody>{this.renderPageContent()}</EuiPageBody>
       </EuiPage>
     );
   }
 }
 
-TableListViewUi.propTypes = {
-  tableColumns: PropTypes.array.isRequired,
-
-  noItemsFragment: PropTypes.object,
-
-  findItems: PropTypes.func.isRequired,
-  deleteItems: PropTypes.func,
-  createItem: PropTypes.func,
-  editItem: PropTypes.func,
-
-  listingLimit: PropTypes.number,
-  initialFilter: PropTypes.string,
-
-  entityName: PropTypes.string.isRequired,
-  entityNamePlural: PropTypes.string.isRequired,
-  tableListTitle: PropTypes.string.isRequired,
-};
-
-TableListViewUi.defaultProps = {
-  initialFilter: EMPTY_FILTER,
-};
-
-export const TableListView = injectI18n(TableListViewUi);
```

Comment:
We should wrap the render tree into an `I18nProvider` if we are removing this - I haven't checked but it looks like there is no I18nProvider anymore for the dashboard listing. But even if not it's easier to use with an `I18nProvider`.

Reply:
@maryia-lapata Did you adress this comment somewhere?

Reply:
Not yet, I'll do it today.

Reply:
@flash1293 I checked this PR with the `zh-CN` locale and as you can see on the screenshot the labels are localized:
![image](https://user-images.githubusercontent.com/11524637/68673396-b208a180-0564-11ea-8cc8-8b0bf1c4fbe5.png)



`injectI18n` is HOC that just passes `intl` object into `props`.
The Dashboard listing is already wrapped into i18nContext here https://github.com/maryia-lapata/kibana/blob/master/src/legacy/core_plugins/kibana/public/dashboard/index.js#L53 . So no need to do it again.